source/core/parsing-and-execution-unit.cpp

```diff
@@ -43,7 +43,7 @@ ParsingAndExecutionUnit::ParsingAndExecutionUnit(
 , _syntaxInformation(_parser->getSyntaxInformation())
 , _setContextInformation([](const std::vector<ContextInformation> &x) {})
 , _setFinalRepresentation([](const FinalRepresentation &x) {})
-, _throwError(([](const std::string &x, const std::vector<std::string> &y) {}))
+, _throwError(([](const Translateable& t) {}))
```

Comment:
These parameters here don't have to be named if they're not used anyway:
`_throwError(([](const Translateable&) {}))` is fine.